Students/Minseong/drinks.js

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient()
+
+const drinks = async(req, res) => {
+    try {
+        const drinks = await prisma.$queryRaw(`
+        SELECT drinks.id, drinks.korean_name, drinks.english_name, categories.name FROM drinks join categories on drinks.category_id = categories.id
+      `)
+
```

Comment:
```suggestion
        const drinks = await prisma.$queryRaw(`
        SELECT
            drinks.id,
            drinks.korean_name,
            drinks.english_name,
            categories.name
        FROM drinks
        JOIN categories 
        ON drinks.category_id = categories.id
      `)
```
SQL문 컨벤션으로는 위와 같이 SQL 명령어 마다 줄을 분리해주시는 것이 가독성 측면에서 좋습니다 :)

Reply:
감사합니다! 수정하겠습니다